gitbook-docs/request_response.md

```diff
@@ -0,0 +1,142 @@
+# Request/Response
+
+## WebSockets and Other Full-duplex Protocols
+
+The exact format of the message for a specific request is defined elsewhere in the specification.
+
+A request should include the `context` when appropriate and must include a client generated `requestId`. The
```

Comment:
I think I asked this earlier, so sorry if I am repeating myself here: does `context` have a function in relation to request/response mechanism? If it is related to the individual, specific request then it is confusing to have it mentioned here in the request/response section.